mapbox_vector_tile/polygon.py

```diff
@@ -196,7 +200,7 @@ def make_valid_multipolygon(shape):
     return MultiPolygon(new_g)
 
 
-def make_it_valid(shape):
+def make_it_valid(shape, asserted=True):
```

Comment:
Does this change the behaviour of `make_it_valid(shape)` that might be called from elsewhere in the code? If that might happen, would it be better to default `asserted=False` so that existing code doesn't change behaviour?